roles/pulp_common/tasks/install_pip.yml

```diff
@@ -93,7 +93,7 @@
 
 - name: Include pulp-rpm prereq role
   include_role:
-    name: pulp_rpm_prerequisites
+    name: pulp.pulp_installer.pulp_rpm_prerequisites
```

Comment:
Is this necessary?

Does it mean that even if I do want to run the roles from the pulp_installer directory directly and I define variables (and possibly create symlinks within the tree), I cannot?

Reply:
Not quite sure, but in my experience the ansible collection concept is all in.
e.g. it's impossible to write a module that works both inside and outside of a collection.
IMHO: We provide (and support) this thing as a collection. If a user wants to go a different way, she is on her own and probably able to figure it out.
It's also quite easy (`make dist`) to build a collection artifact you can install with your local changes.

Reply:
> IMHO: We provide (and support) this thing as a collection. If a user wants to go a different way, she is on her own and probably able to figure it out.

I agree! We have to limit our scope

Reply:
Turns out, this _is_ needed. To that point i believe it's not only needed for our CI, but whenever you use the installer as a collection with ansible 2.8.
see https://github.com/pulp/pulp_installer/pull/382/checks?check_run_id=975603243